save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Vulnerability.kt

```diff
@@ -0,0 +1,25 @@
+package com.saveourtool.save.entities
+
+import com.saveourtool.save.spring.entity.BaseEntity
+import javax.persistence.Entity
+
+/**
+ * @property name
+ * @property progress
+ **/
+@Entity
+class Vulnerability(
+
+    var name: String,
+
+    var progress: Int,
+
+) : BaseEntity() {
```

Comment:
can you please extend `BaseEntityWithDto` here?

Reply:
Yes, sure.